[2025-10-02T08:00:08.782+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T08:00:08.787+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sentiment140_end2end_single.sf_check_label_dist manual__2025-10-02T07:59:02.866757+00:00 [queued]>
[2025-10-02T08:00:08.790+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sentiment140_end2end_single.sf_check_label_dist manual__2025-10-02T07:59:02.866757+00:00 [queued]>
[2025-10-02T08:00:08.790+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-02T08:00:08.794+0000] {taskinstance.py:2330} INFO - Executing <Task(SQLExecuteQueryOperator): sf_check_label_dist> on 2025-10-02 07:59:02.866757+00:00
[2025-10-02T08:00:08.797+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1036) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T08:00:08.798+0000] {standard_task_runner.py:63} INFO - Started process 1039 to run task
[2025-10-02T08:00:08.798+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sentiment140_end2end_single', 'sf_check_label_dist', 'manual__2025-10-02T07:59:02.866757+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/edaproject.py', '--cfg-path', '/tmp/tmpgd_dt0hw']
[2025-10-02T08:00:08.799+0000] {standard_task_runner.py:91} INFO - Job 104: Subtask sf_check_label_dist
[2025-10-02T08:00:08.820+0000] {task_command.py:426} INFO - Running <TaskInstance: sentiment140_end2end_single.sf_check_label_dist manual__2025-10-02T07:59:02.866757+00:00 [running]> on host 9d9a5d472595
[2025-10-02T08:00:08.849+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sentiment140_end2end_single' AIRFLOW_CTX_TASK_ID='sf_check_label_dist' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T07:59:02.866757+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-02T07:59:02.866757+00:00'
[2025-10-02T08:00:08.849+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T08:00:08.853+0000] {sql.py:276} INFO - Executing: 
            USE ROLE ACCOUNTADMIN;
            USE WAREHOUSE COMPUTE_WH;
            USE DATABASE YOUR_DB;
            USE SCHEMA PUBLIC;
            SELECT LABEL, COUNT(*) AS CNT
            FROM YOUR_DB.PUBLIC.SENTIMENT140_CLEAN
            GROUP BY 1
            ORDER BY 1;
        
[2025-10-02T08:00:08.857+0000] {base.py:84} INFO - Using connection ID 'snowflake_default' for task execution.
[2025-10-02T08:00:08.860+0000] {base.py:84} INFO - Using connection ID 'snowflake_default' for task execution.
[2025-10-02T08:00:08.860+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-02T08:00:08.861+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-02T08:00:10.673+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T08:00:10.674+0000] {sql.py:470} INFO - Running statement: USE ROLE ACCOUNTADMIN;, parameters: None
[2025-10-02T08:00:10.869+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T08:00:10.870+0000] {sql.py:479} INFO - Rows affected: 1
[2025-10-02T08:00:10.871+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-10-02T08:00:10.872+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6fc0-0306-92f7-0000-00131e67f029
[2025-10-02T08:00:10.872+0000] {sql.py:470} INFO - Running statement: USE WAREHOUSE COMPUTE_WH;, parameters: None
[2025-10-02T08:00:11.062+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T08:00:11.063+0000] {sql.py:479} INFO - Rows affected: 1
[2025-10-02T08:00:11.064+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-10-02T08:00:11.065+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6fc0-0306-8f88-0000-00131e67d1e5
[2025-10-02T08:00:11.065+0000] {sql.py:470} INFO - Running statement: USE DATABASE YOUR_DB;, parameters: None
[2025-10-02T08:00:12.036+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T08:00:12.037+0000] {sql.py:479} INFO - Rows affected: 1
[2025-10-02T08:00:12.038+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-10-02T08:00:12.038+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6fc0-0306-8fe9-0000-00131e677169
[2025-10-02T08:00:12.038+0000] {sql.py:470} INFO - Running statement: USE SCHEMA PUBLIC;, parameters: None
[2025-10-02T08:00:12.211+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T08:00:12.212+0000] {sql.py:479} INFO - Rows affected: 1
[2025-10-02T08:00:12.212+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-10-02T08:00:12.213+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6fc0-0306-8fe9-0000-00131e67716d
[2025-10-02T08:00:12.213+0000] {sql.py:470} INFO - Running statement: SELECT LABEL, COUNT(*) AS CNT
            FROM YOUR_DB.PUBLIC.SENTIMENT140_CLEAN
            GROUP BY 1
            ORDER BY 1;, parameters: None
[2025-10-02T08:00:12.996+0000] {cursor.py:1149} INFO - Number of results in first chunk: 2
[2025-10-02T08:00:12.997+0000] {sql.py:479} INFO - Rows affected: 2
[2025-10-02T08:00:12.997+0000] {snowflake.py:407} INFO - Rows affected: 2
[2025-10-02T08:00:12.997+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6fc0-0306-904a-0000-00131e676185
[2025-10-02T08:00:13.302+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T08:00:13.303+0000] {connection.py:762} INFO - closed
[2025-10-02T08:00:13.491+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2025-10-02T08:00:13.552+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T08:00:13.597+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sentiment140_end2end_single, task_id=sf_check_label_dist, run_id=manual__2025-10-02T07:59:02.866757+00:00, execution_date=20251002T075902, start_date=20251002T080008, end_date=20251002T080013
[2025-10-02T08:00:13.660+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T08:00:13.679+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-02T08:00:13.680+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
